src/UIState/NumberCollectorState.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * Abstract superclass for the NumberCollector UI states
+ */
+
+#pragma once
+#include "UIState.h"
+
+class NumCollectorState : public UIState {
+public:
+  NumCollectorState() {
+    LiquidCrystal_TC::instance()->writeLine(" ", 1);
```

Comment:
Does this replace the entire line or only the first character? We might need to write sixteen spaces.